index.html

```diff
@@ -94,7 +94,7 @@ <h4>QuickLook Support</h4>
         <div class="row justify-content-between text-center text-md-left">
           <div data-aos="fade-right" data-aos-duration="1000" class="col-md-6">
             <h2 class="font-weight-bold">Take a look inside</h2>
-            <p class="my-4">Visit the Wiki to find tips, tricks and to learn more about Files</p>
+            <p class="my-4">Visit the Wiki to find tips and tricks and to learn more about Files</p>
```

Comment:
Perhaps `&` is more suitable here? 🤔 

```
Visit the Wiki to find tips & tricks and to learn more about Files
```

any thoughts, @yaichenbaum?

Reply:
@5E7EN I think that `&` is better here.